docs/sources/whatsnew/whats-new-in-v9-3.md

```diff
@@ -0,0 +1,75 @@
+---
+_build:
+  list: false
+aliases:
+  - /docs/grafana/latest/guides/whats-new-in-v9-3/
+description: Feature and improvement highlights for Grafana v9.3
+keywords:
+  - grafana
+  - new
+  - documentation
+  - '9.3'
+  - release notes
+title: What's new in Grafana v9.3
+weight: -33
+---
+
+# What’s new in Grafana v9.3 (Beta)
+
+Welcome to Grafana v9.3. If you’d prefer to dig into the details, check out the complete [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
```

Comment:
```suggestion
Welcome to Grafana v9.3. This document provides an overview of what's new in this release. For more detail, refer to the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
```

Reply:
How about "Welcome to Grafana 9.3. Read the What's New to learn all about the latest release."

Maybe a little more chatty and less formal, directly addressing the user...?

(eventually here we can add: prepare for upgrade, upgrade..?

Reply:
I like it, thanks.